README.rst

```diff
@@ -81,12 +81,17 @@ are also encountered in the wild. Other less common or virtually
 unheard of flag combinations (such as ``--with-pydebug`` (``d``) and
 ``--without-pymalloc`` (absence of ``m``)) are not provided.
 
-It'd be good to put an example of how to use these images here, but
-that isn't written yet. If you want to know, then bug us on the
-mailing list to fill in this section :-). However, one useful tip is
-that a list of all interpreters can be obtained with
-``/opt/python/*/bin/python``.
-
+Example
+-------
+An example project which builds 32- and 64-bit wheels for each Python interpreter
+version can be found here: https://github.com/pypa/python-manylinux-demo.
+
+This demonstrates how to use these docker images in conjunction with auditwheel
+to build manylinux-compatible wheels. NB: for the 32-bit images running on a 64-bit
```

Comment:
...wheels using the free [travis-ci](https://travis-ci.org) continuous integration service.